core/src/main/java/org/elasticsearch/index/settings/IndexDynamicSettingsModule.java

```diff
@@ -86,6 +85,7 @@ public IndexDynamicSettingsModule() {
         indexDynamicSettings.addDynamicSetting(IndexingSlowLog.INDEX_INDEXING_SLOWLOG_THRESHOLD_INDEX_TRACE, Validator.TIME);
         indexDynamicSettings.addDynamicSetting(IndexingSlowLog.INDEX_INDEXING_SLOWLOG_REFORMAT);
         indexDynamicSettings.addDynamicSetting(IndexingSlowLog.INDEX_INDEXING_SLOWLOG_LEVEL);
+        indexDynamicSettings.addDynamicSetting(IndexingSlowLog.INDEX_INDEXING_SLOWLOG_SOURCE_TO_LOG);
```

Comment:
oh sorry, I was not reading the diff correctly, it looked to me like setting sourceToLog was part of the constructor. Nevermind :)